packages/integration-tests/tests/bridge_erc20.ts

```diff
@@ -59,12 +62,103 @@ describe('Bridging ERC20 tokens', async function () {
   // override the default test timeout from 2000ms
   this.timeout(DEFAULT_TIMEOUT_MS);
 
+  async function fetchTransaction(
+    fuel_bridge: BridgeFungibleToken,
+    fuelTokenSender: FuelWallet,
+    to: string,
+    NUM_TOKENS: bigint,
+    DECIMAL_DIFF: bigint,
+    useMessageCoin: boolean
+  ): Promise<ScriptTransactionRequest> {
+    if (useMessageCoin) {
+      const tx = await fuel_bridge.functions
+        .withdraw(to)
+        .addContracts([fuel_bridge, fuel_bridgeImpl])
+        .txParams({
+          tip: 0,
+          maxFee: 1,
+        })
+        .callParams({
+          forward: {
+            amount: new BN(NUM_TOKENS.toString()).div(
+              new BN(DECIMAL_DIFF.toString())
+            ),
+            assetId: fuel_testAssetId,
+          },
+        });
+
+      // verify the incoming messages generated when base asset is minted on fuel
+      let incomingMessagesonFuel = await env.fuel.signers[0].getMessages();
+
+      expect(incomingMessagesonFuel.messages.length === 1);
+      expect(
+        incomingMessagesonFuel.messages[0].amount === fuels_parseEther('1')
+      );
```

Comment:
```suggestion
      expect(incomingMessagesonFuel.messages.length >= 1);
```

Is there a reason why you need that the amount and length are exactly one?

Reply:
I bridge 1 eth at the start of the test, so that was the only reason to have a strict equality